resources/PythonPip3Dsc/PythonPip3Dsc.psm1

```diff
@@ -106,7 +177,10 @@ function Get-PackageNameWithVersion
         [string]$Version,
 
         [Parameter()]
-        [switch]$IsUpdate
+        [switch]$IsUpdate,
+
+        [Parameter()]
+        [switch]$DryRun
```

Comment:
Does Get-PackageNameVersion need a DryRun?

Reply:
Clarified now in function. It is used in the helper functions inside the class.